source/docs/tag-plugins.md

```diff
@@ -278,6 +278,23 @@ For instance:
 
 {% post_link 2018-10-19-hexo-3-8-released 'Link to a post' %}
 
+The text of the post's title and the custom title is escaped by default. You can use the `escape` option to disable escaping.
```

Comment:
"Post's title and custom text are escaped by default." The rest is fine.
also move the line next to "You can customize the text to display,..." paragraph, otherwise it looks like it's under "Display custom text." section.

Reply:
Updated.